config/currency_iso.yml

```diff
@@ -1446,19 +1446,20 @@ mop:
   thousands_separator: ","
   iso_numeric: '446'
   smallest_denomination: 10
-mro:
+mru:
   priority: 100
-  iso_code: MRO
-  name: Mauritanian Ouguiya
+  iso_code: MRU
+  name: Mauritanian New Ouguiya
   symbol: UM
+  disambiguate_symbol: MRU
   alternate_symbols: []
```

Comment:
Shouldn't we add `MRO` here? (I have no context at all on how  this works and how it's used, just remind of country_db's `alternate_names`)

Reply:
> Shouldn't we add MRO here?

Ah, but MRO is not an alternate name for MRU.
MRO is the [original Mauritanian Ouguiya](https://en.wikipedia.org/wiki/Mauritanian_ouguiya#First_Ouguiya_(MRO)) (1973-2017).
MRU is the [second (or "new") Mauritanian Ouguiya](https://en.wikipedia.org/wiki/Mauritanian_ouguiya#Second_ouguiya_(MRU)) (2018-present).
1 MRU == 10 MRO
MRO is listed separately in `config/currency_historic.yml`.